homeassistant/components/modbus/switch.py

```diff
@@ -184,6 +194,49 @@ def available(self) -> bool:
         """Return True if entity is available."""
         return self._available
 
+    def _read_modbus(self, address) -> int | None:
```

Comment:
It is very hard to do a review when you move code around (at least that is my impression), please try to make your changes by touching as little as possible in the original code.

When you move code around, we need to look carefully at the moved code to see if it have changed, also later the maintainer will look at the old (but moved code) and see it as new code...meaning they might want changes according to the standards used today.